src/config/apprunner.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "APP_RUNNER",
+  "displayName": "App Runner",
+  "rules": [
+    {
+      "aws": {
+        "logGroup": "/aws/apprunner/%{DATA:resource_id}/%{DATA:service_id}/%{GREEDYDATA:log_type}"
+      },
+      "sources": [
+        {
+          "sourceType": "logs",
+          "source": "log_group",
+          "condition": "$prefix('/aws/apprunner/')"
+        }
+      ],
+      "attributes": [
+        {
+          "key": "aws.service",
+          "pattern": "'apprunner'"
+        },
+        {
+          "key": "aws.resource.id",
+          "pattern": "resource_id"
+        },
+        {
+          "key": "aws.arn",
+          "pattern": "format_arn('arn:{}:apprunner:{}:{}:service/{}/{}', [partition, region, account_id, resource_id, service_id])"
+        },
+        {
+          "key": "severity",
+          "pattern": "if( starts_with(log_content, 'ERROR'), &'ERROR', &if( starts_with(log_content, 'WARN'), &'WARN', &'INFO', @), @)"
```

Comment:
Minor, I'd have a test case for it. Mostly as a doc of what we have handled. 
I wanted to see the logs in the jxw, but can't find them, I suppose you have them in some other env.
Major, thank you for all the examples;D